tarynmartin/personal-portfolio
# Reorganized and restyled landing page

#### What’s this PR do?
- Changed up layout, background images, and general styling
- Added some additional hover effects

#### Where should the reviewer start?
- LandingPage.js to see changes
- Main.js to see where navigation buttons have been placed
- App.js to see added route
- Styles.js to see changes to styling 

#### How should this be manually tested?
- in terminal, run npm start
- in browser, click on name to see navigation buttons
- click on buttons to see continuing functionality
